### ISSUE TITLE
Error: You must provide sources for reading API files.

### ISSUE THREAD

**Issue by mits87** [Surnet/swagger-jsdoc#233]
Hi, 

I have a small problem.
Right now when I'm trying to do some config definition:

```
{
  swaggerDefinition: {
    openapi: '3.0.3',
    info: {
      title: 'Title',
      version: '0.1.1',
      description: 'Description',
    },
    host: 'localhost:3000',
    basePath: '/',
  },
  apis: ['./src/handlers/**/*.js'],
}
```

and run the CLI command, I'm getting an error:

```
You must provide sources for reading API files.
Either add filenames as arguments, or add an "apis" key in your definitions file.
```

The error appears because there is no `apis` key inside `swaggerDefinition` key which is IMHO wrong. 
Can you confirm and fix it?

**Comment by kalinchernev** [comment]
Hi @mits87 you are very right, the `apis` shouldn't be provided through the swaggerDefinition object, but for some reason they've been introduced as an [option in the CLI](https://github.com/Surnet/swagger-jsdoc/blob/master/bin/swagger-jsdoc.js#L65)

It's one of the things I would be happy to accept pull requests. I am even ok with it being a breaking change in 6.0. 

I haven't had the time to write it, but if you have the opportunity:
- we'll need to first remove the check https://github.com/Surnet/swagger-jsdoc/blob/master/bin/swagger-jsdoc.js#L65
- we'll have to update https://github.com/Surnet/swagger-jsdoc/blob/master/bin/swagger-jsdoc.js#L78 too to not expect swaggerDefinition.apis
- regarding tests https://github.com/Surnet/swagger-jsdoc/blob/master/test/cli.spec.js it might be there are no breaking such as there is no case for this, though if you have the opportunity to write one for the case it will be highly appreciated
- lastly, a small doc update to remove the apis mention from https://github.com/Surnet/swagger-jsdoc/blob/master/docs/CLI.md#definition-file and remove `When you don't want to or can't pass apis from the definition above, specify like following:`

Thanks for bringing up the subject!

**Comment by mits87** [comment]
The PR: https://github.com/Surnet/swagger-jsdoc/pull/234

**Comment by kalinchernev** [comment]
Many thanks! Will review at first opportunity!

**Comment by kalinchernev** [comment]
@mits87 I think I have gotten a bit tired during the week and didn't pay enough attention. I merged your pull request and closed the issue, but the issue is not solved.

With the state of the latest release, the CLI got its input api files required. I made a few tweaks in the docs and just now realized I had to be more careful in the review.

The `apis` of options will not be respected from the CLI, only the parameters
